pkg/network/protocols/telemetry/delta.go

```diff
@@ -66,14 +64,6 @@ func (c *clientState) ValueFor(m *Metric) int64 {
 	defer c.mux.Unlock()
 
 	prev := c.prevValues[name]
-	if prev > current {
-		// let the library client know if this metric is being misconfigured
-		log.Debugf(
-			"error: metric %q is not growing monotonically but it was instantiated with `OptMonotonic`",
-			name,
-		)
-	}
-
```

Comment:
why did you remove the check?

Reply:
Because counters now are always monotonic